location/test_helpers.py

```diff
@@ -45,14 +51,39 @@ def create_test_location(loc_type, valid=True, custom_props=None):
 
     
     
-def create_test_village(custom_props=None):
-    if custom_props is None:
-        custom_props = {}
-    custom_props["name"]=custom_props['name'] if "name" in custom_props else "Test Village"
-    test_region = create_test_location('R',custom_props={"name": "Region" + custom_props["name"]})
-    test_district = create_test_location('D', custom_props={"parent": test_region,"name":"District " + custom_props["name"]})
-    test_ward = create_test_location('W', custom_props={"parent": test_district,"name":"Ward" + custom_props["name"]})
-    custom_props["parent"]=test_ward
+def create_test_village(custom_props={}):
```

Comment:
You cannot do that, in python default parameters are initialize once, for mutable it means that they might keep a value between call, yes it is weird but I saw it happening myself this week

Reply:
You are right. Good catch. Fixed.